src/transaction-flow/input/RevokePermissions/RevokePermissions-flow.tsx

```diff
@@ -20,6 +21,7 @@ import { dateTimeLocalToDate, dateToDateTimeLocal } from '@app/utils/datetime-lo
 
 import { ControlledNextButton } from './components/ControlledNextButton'
 import { GrantExtendExpiryView } from './views/GrantExtendExpiryView'
+import { NameConfirmationWarningView } from './views/NameConfirmationWarningView'
 import { ParentRevokePermissionsView } from './views/ParentRevokePermissionsView'
```

Comment:
<img width="683" alt="Screenshot 2024-04-15 at 1 25 37 PM" src="https://github.com/ensdomains/ens-app-v3/assets/1967258/1cd508f4-82fa-4770-ad35-4d2cfcde03e0">

Grammar: "You are making changes that can only be undone **when** the name expires."

Reply:
pardon my french